### DISCUSSION TITLE
https://vlaaad.github.io/all-i-want-from-json

### DISCUSSION THREAD

**Discussion by giscus** [vlaaad/vlaaad.github.io#21] [Posts]
# https://vlaaad.github.io/all-i-want-from-json

Following the recent uptick of JSON-related discussions, here is my opinion on the matter

https://vlaaad.github.io/all-i-want-from-json

**Comment by dislux-hapfyl** [comment]
All I want is Python without indentation...
0dc MyTextBox Frame

1df s __init__ parent

2fc l "super().__init__" parent

2va s master parent

2fc s configure bg=$BG

2vf s this_text_box Text $TEXT_PROP

2fc s this_text_box.pack $TEXT_PACK

2vf s words_var StringVar

2fc s words_var.set '"0 | 1"'

https://nopaste.net/shy.sh

https://nopaste.net/shy-editor.shy

source shy.sh ; bash shy-editor.shy | python3
